semcore/card/src/Card.jsx

```diff
@@ -39,21 +40,31 @@ function Title(props) {
   const { styles, innerHint, Children, innerHintAriaLabel, hintAfterAriaLabel } = props;
   const hintAfter = props.hintAfter || props.hint;
   const STitle = Root;
-  const SIcon = InfoM;
+  const SIcon = ButtonLink;
   const STooltip = DescriptionTooltip;
   return sstyled(styles)(
     <>
       <STitle render={Text}>
         <Children />
         {innerHint && (
-          <STooltip ml={1} title={innerHint} aria-label={innerHintAriaLabel}>
-            <SIcon interactive />
+          <STooltip ml={1} title={innerHint}>
```

Comment:
Noticed 2 issues with innerHint and hintAfter:

1. The icon is a bit too low
2. There's one redundant empty tooltip appearing on hover

<img width="749" alt="image" src="https://github.com/user-attachments/assets/126ef40c-52cd-4331-aa72-ee04959bf240">

Reply:
> The icon is a bit too low
This is because of vitepress styles, I've fixed it

> The aria-label appears as a Hint on hover, but it shouldn't.